acceptance/analyzer_test.go

```diff
@@ -206,13 +206,28 @@ func testAnalyzerFunc(platformAPI string) func(t *testing.T, when spec.G, it spe
 		})
 
 		when("the provided layers directory isn't writeable", func() {
+			var imageName, authConfig string
+			var err error
```

Comment:
nit: You could move the `err` declaration inside the `it.Before`, as it's not needed in other scopes.

Reply:
Fixed in the following [commit](https://github.com/buildpacks/lifecycle/pull/646/commits/8a82acf4f285ed83dbfb346b7f65f106b548d8aa)